pkg/pillar/hypervisor/kvm.go

```diff
@@ -333,7 +339,7 @@ func newKvm() Hypervisor {
 			ctrdContext:  *ctrdCtx,
 			devicemodel:  "virt",
 			dmExec:       "/usr/lib/xen/bin/qemu-system-aarch64",
-			dmArgs:       []string{"-display", "none", "-S", "-no-user-config", "-nodefaults", "-no-shutdown", "-overcommit", "mem-lock=on", "-overcommit", "cpu-pm=on", "-serial", "chardev:charserial0"},
+			dmArgs:       []string{"-display", "none", "-S", "-no-user-config", "-nodefaults", "-no-shutdown", "-overcommit", "mem-lock=on", "cpu-pm=on", "-serial", "chardev:charserial0"},
```

Comment:
Hey @temerkhanov is removing -overcommit required for the USB passthrough? I don't think it is and as such even if we do it -- lets put it in a separate PR.

Reply:
There is one more "overcommit" parameter, right before "mem-lock-on". Do we really want it set twice?

Reply:
I don't think qemu command line parsing works in a way that would allow you to skip the -overcommit before `cpu-pm=on`. The most plausible scenario is that `-overcommit mem-lock=on,cpu-pm=on` will work but even that should go into a separate commit since this kind of clean up doesn't seem to have anything to do with enabling USB passthrough @temerkhanov

Reply:
Removed the commit it question